ansible/roles/screenly/tasks/main.yml

```diff
@@ -57,6 +57,12 @@
     owner: root
     group: root
 
+- cron:
+    name: Cleanup screenly_assets
+    special_time: reboot
```

Comment:
Nodes can have weeks or months uptime, so let's run this at say 1AM every night instead.